src/kernl/model_optimization.py

```diff
@@ -22,15 +22,11 @@
 from kernl.optimizer.dynamo_backend import dynamo_backend_ofi
 
 
-# single shared pool by default
-_pool: (int, int) = torch.cuda.graph_pool_handle()
```

Comment:
it will be covered here:
- https://github.com/ELS-RD/kernl/issues/205
- https://github.com/ELS-RD/kernl/issues/206
(my guess is that both will be managed at the same time)